src/components/pages/Table/SearchBar.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  inputChange,
+  filterSearch,
+  filterCategories,
+  resetFilter,
+} from '../../../state/actions';
+
+function SearchBar(props) {
+  const {
+    asylum,
+    categories,
+    filteredData,
+    filteredCount,
+    filterCategories,
+    filterSearch,
+    form,
+    inputChange,
+    resetFilter,
+  } = props;
+
+  function onChange(evt) {
+    const { name, value } = evt.target;
+    const formValues = { name, value };
+    inputChange(formValues);
+  }
+
+  function onSubmit(evt) {
+    evt.preventDefault();
+
+    const data = filteredCount === 0 ? asylum : filteredData;
+    const searchTerm = form.searchTerm;
+    const searchCategory = form.category;
+
+    if (searchTerm === '' || searchCategory === '') {
+      alert('Please Enter A Search Term & Filter Category.');
+    } else {
+      filterSearch({ data, searchTerm, searchCategory });
+      filterCategories(categories, searchCategory);
+    }
+  }
+
+  function reset(evt) {
+    evt.preventDefault();
+    resetFilter();
+  }
+
+  return (
+    <form id="searchBar">
+      <label>
+        {' '}
+        Search Term
+        <input
+          id="searchTerm"
+          name="searchTerm"
+          type="text"
+          value={form.searchTerm}
+          onChange={onChange}
+        ></input>
+      </label>
+      <select
+        onChange={onChange}
+        value={form.category}
+        name="category"
+        id="category"
+      >
+        <option value="">Pick A Category</option>
```

Comment:
Can I ask what the importance of more than one category at one search point is over the filtering process now? For instance, if searching for a specific asylum office and a specific ethnicity/race, what would be the point of entering more than one search category at a time? 

Unless there are also plans for a second pop-up search bar to take in that second search value as well. Otherwise, I believe it would just be too complicated for regular asylum users. 

If this is to account for things like the hidden parameters, I believe those could be added at a later date when we have our full official data would be.